### ISSUE TITLE
Functionality inaccessible for keyboard only and screen reader user

### ISSUE THREAD

**Issue by mkdir-washington-edu** [hypothesis/product-backlog#1639]
From the HighWire Press / McGraw-Hill Accessibility spreadsheet: https://hypothes-is.slack.com/archives/C8TPC8XMK/p1736535764817829

**Highwire Press Reproduction**
1. Using keyboard, try to select any text on the page.
2. Observe that this functionality is inaccessible for keyboard users.

**Highwire Expected Result**
The text selection functionality should be accessible for all users.

**Highwire Actual Result**
Keyboard users cannot select the text content making the functionality inaccessible for keyboard users. Additionally, options such as "Annotate" and "Highlight" become available only when users select text using a mouse and not accessible via keyboard.

**Highwire Rationale**
The functionality of controls that dynamically add or update content on the page should be indicated to screen reader users. This will enable screen reader users to interact with such controls effectively and access the added or updated content.
When static content is displayed on a page, screen reader can read the information for their users. However, when content is added dynamically, screen readers will not know that there is new content available on the page. In addition, screen readers will not know where the new content is displayed and thus will not be able to inform about the same to their users.

**Highwire Recommendation**
The text options that become available when users select any text using the mouse must also be available on focus when users navigate the content through the keyboard.
Apply the following changes:
• Provide users with options to select text using the keyboard. Add keyboard shortcuts to enable users to quickly turn On/Off the selection mode. Additionally, provide visual instructions about the keyboard shortcuts.
• Modify the script to include keyboard specific event handlers ""onfocus"" along with ""onhover"" to make the functionality available for keyboard users.
• When the text options are displayed, set the keyboard focus to the first option i.e. ""Annotate"" button (tabindex=""-1"" can be used for this purpose).
